### ISSUE TITLE
Update ember dependencies of packages

### ISSUE THREAD

**Issue by jenweber** [cardstack/cardstack#748]
Breaking this out from the suggestion in #741

**Comment by ef4** [comment]
Yes, this would be great.

ember-cli-update is the best way to do upgrades, but last I checked it didn't understand monorepos. Maybe that has been fixed, or maybe we can fix it.

**Comment by jenweber** [comment]
It does not yet support a project like this one:

```
ember-cli-update --to 3.8.1
Ember CLI project type could not be determined
```

In the meantime, I have a short bash script that will check if a package is an addon and update it. `ember-cli-update` will only operate on a clean git state though, so step one of contributing back might be to add an option that allows it to keep running with uncommitted changes.

**Comment by jenweber** [comment]
The core-types repo has [this issue](https://github.com/thefrontside/ember-let/issues/49) and something else going on.

**Comment by jenweber** [comment]
For future reference - ember-elsewhere blocks were not rendering at all after the update. This was due to https://github.com/ef4/ember-elsewhere/issues/24 and things worked again following updating that dependency.

**Comment by jenweber** [comment]
Script to check package versions:

```sh
for dir in packages/*     # list directories in the form "/tmp/dirname/"
    do
        if [ -d ./${dir}/addon ]; then
            echo "---------------------------------------------"
            ( cd ./${dir} && echo ${dir} && ember -v)
        fi
    done
```

**Comment by jenweber** [comment]
remaining to-do
- [ ] image
- [ ] core-types
- [ ] card-picker

**Comment by jenweber** [comment]
Closed by #769 
Moving remaining packages to separate issues.
